atlp-rwanda/rca-phantom-team1-bn
# Chore: added ci-cd

## CI/CD PR

Configure a continuous integration pipeline using GitHub Actions.

It contains the following acceptance criteria:

1. Configured Testing environment with Mocha, Chai and Sinon to ensure all functionalities are tested before deployment.

2. GitHub Actions is configured as a continuous integration service to build and test the app.

3. GitHub Actions badge has been added to your repo.

## How to manually test it 

Run **npm test** to see if all your tests are working as expected.

<img width="737" alt="image" src="https://user-images.githubusercontent.com/50233828/223964050-76d3c604-6d10-4f0c-a69f-469fc6187292.png">

<img width="752" alt="image" src="https://user-images.githubusercontent.com/50233828/223964360-4c8f5027-c255-44c4-8be6-7646c146b683.png">

To check if the GitHub Actions badge has been added to your repo, go to **README** File.

<img width="620" alt="image" src="https://user-images.githubusercontent.com/50233828/223965069-0470e57c-4abe-40fb-ac72-4cdb516fcb07.png">
